webapp/src/app.tsx

```diff
@@ -3539,7 +3545,7 @@ export class ProjectView
     }
 
     completeTutorialAsync(): Promise<void> {
-        pxt.tickEvent("tutorial.complete");
+        pxt.tickEvent("tutorial.complete", { tutorial: this.state.header?.tutorial?.tutorial });
```

Comment:
@shakao can we add this tutorial custom dimension to all the following ticks? 

tutorial.step
tutorial.start
tutorial.showhint
tutorial.shared
tutorial.previous
tutorial.next
tutorial.hint.next
tutorial.faileddecompile
tutorial.exit
tutorial.complete
menu.exittutorial
hash.tutorial